EIPS/eip-4200.md

```diff
@@ -39,17 +39,21 @@ We introduce two new instructions on the same block number [EIP-3540](./eip-3540
 
 1. `RJUMP` (0x5c)
 2. `RJUMPI` (0x5d)
+3. `RJUMPV` (0x5e)
 
-If the code is legacy bytecode, both of these instructions result in an *exceptional halt*. (*Note: This means no change to behaviour.*)
+If the code is legacy bytecode, all of these instructions result in an *exceptional halt*. (*Note: This means no change to behaviour.*)
 
 If the code is valid EOF1:
 
-1. `RJUMP relative_offset` (0x5c), sets the `PC` to `PC_post_instruction + relative_offset`.
-2. `RJUMPI relative_offset` (0x5d), pops a value (`condition`) from the stack, and sets the `PC` to `PC_post_instruction + ((condition == 0) ? 0 : relative_offset)`.
+1. `RJUMP relative_offset` sets the `PC` to `PC_post_instruction + relative_offset`.
```

Comment:
I cannot find where this `PC_post_instruction` is defined.

Reply:
On L52.